cmd/gen-go-sample/main.go

```diff
@@ -353,14 +353,38 @@ func (g *generator) genSample(sampConf schema_v1p2.Sample, methConf GAPICMethod)
 
 	p := g.pt.Printf
 
-	// Region tag, function signature and new client
+	// Region tag
 	argStr, err := argListStr(initInfo, g)
 	if err != nil {
 		return err
 	}
 	p("")
 	p("// [START %s]", sampConf.RegionTag)
 	p("")
+
+  // comments
+  emptyArr := make([]string, 0)
+  requiresNewLine := false
+  
+  if sampConf.Description != "" {
+  	writeComment(sampConf.Description, emptyArr, g)
+  	requiresNewLine = true
+  }
+
+  for i, argName := range initInfo.argNames {
+  	comment := initInfo.argTrees[i].comment
+  	if comment == "" {
+  		continue
+  	}
+  	if requiresNewLine {
+  		writeComment("", emptyArr, g)
+  	}
+  	requiresNewLine = true
```

Comment:
so this never gets set back to `false`, meaning that after a single comment, a new line will precede the next comment. Is my understanding correct? Is it *just* a new line for the next thing or a blank line? What if the comment has a new line at the end, does it double up? Or why couldn't we write the comment string and append a newline to it before writing? This just feels a little wonky to me and I'd like to talk it out.

Reply:
It is a blank line between the method comment and each parameter comment.

> What if the comment has a new line at the end, does it double up?
I think it shouldn't, mostly because multi-line strings in YAML always have a newline character in the end, but users don't want that to actually result in a new line. However, I apparently missed this case, and I'll add it shortly.

> Or why couldn't we write the comment string and append a newline to it before writing?
We can't append a newline to the last comment, that will cause an empty comment line between the last comment and the function signature.

Reply:
> I think it shouldn't...I'll add it shortly.

sgtm

> We can't append a newline to the last comment, that will cause an empty comment line between the last comment and the function signature.

Ack. There might be a more _explicit_ way to do this (i.e. break before the last item and write it immediately following the loop), but it would be a few more lines of code and it doesn't matter all that much.

Reply:
> break before the last item and write it immediately following the loop...
The problem is that not all parameters have comments. So this would require two passes through all the parameters: the first to find all parameters with comments, and the second generate them.